### PR TITLE
Leafcutterlabs

### PR DESCRIPTION
**Pull request by LeafCutterLabs**
## Description

Correcting/removing the tap dance error from configurator

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Configurator was giving an error due to the use of Tap Dance, the tap dance button was removed

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — CHANGES_REQUESTED** [review]
This doesnt actually have any changes in comparison to master. I would recommend you do not reuse the same branch as before, as that is part of the issue here.

**Review by LeafCutterLabs — COMMENTED** [review]
remove tap dance

File: keyboards/leafcutterlabs/bigknob/rules.mk [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes        # Enable rotary encoders
+TAP_DANCE_ENABLE = no
```
Comment (fauxpark) [inline keyboards/leafcutterlabs/bigknob/rules.mk:24]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by LeafCutterLabs — COMMENTED** [review]
removed tap dance

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Looks like you have some merge conflicts here.

**Review by LeafCutterLabs — COMMENTED** [review]
removing/resolving conflicts

**Review by LeafCutterLabs — COMMENTED** [review]
trying tp pass test

**Review by LeafCutterLabs — COMMENTED** [review]
local compiles fine but fails travis?

**Comment by fauxpark** [comment]
Tap Dance should not be enabled at the keyboard level, since it expects custom code to be present in the keymap, but the Configurator does not use the default keymap, where it would be defined.

**Review by fauxpark — APPROVED** [review]

**Pull request merged**